recipe/session/recipe.go

```diff
@@ -25,7 +25,6 @@ import (
 	"github.com/supertokens/supertokens-golang/recipe/session/api"
 	"github.com/supertokens/supertokens-golang/recipe/session/claims"
 	"github.com/supertokens/supertokens-golang/recipe/session/errors"
-	"github.com/supertokens/supertokens-golang/recipe/session/sessionwithjwt"
```

Comment:
Are there references left for this file? I think it can be removed

Reply:
Ill be refactoring this later (trying to keep the impact of individual PRs down)